### ISSUE TITLE
Can't open ts3video

### ISSUE THREAD

**Issue by Nion098** [mfreiholz/ts3video#50]
When i click right click to enter the ts3video, its not showing, i have even tried to reinstall TS and the plugin, even tried with 32 bit of Team speak, still doesn't show up.

**Comment by mfreiholz** [comment]
Can you provide me with some basic information about your hardware, please?

Make sure that your Windows + Drivers run on the newest version with all updates.

**Comment by Nion098** [comment]
I have Windows 7 professional 64bit. Everything is up to date, i even uninstall everything. I tried with older version, it is working and i am using older version to chat, but it does not work with the new version.

**Comment by jedi404** [comment]
one of my members had this happen on a 64 bit machine with the new version
of TS3 installed. his solution was to completely uninstall and roll back to
the last version of both the plugin and TS3. ive also had users installs of
.11 not complete cleanly and have had to have copied versions of the up to
date .dlls placed in their appdata folders to get the plugin to function
correctly again. the plugin also seems to not like being on a secondary
screen. several users myself included have noted this. the plugin wont load
if theres anyone in a room if its loading on a secondary screen. it just
white screens. also if you start it on the primary but move it later
clients joining locks the application up entirely.  the only workaround
weve found for this is to keep it on the primary screen at all times.  i
can contact you elsewhere if youd like to know more of the bugs my group
has found, or if youd like a broader range of machines to do testing on new
builds.

**Comment by mfreiholz** [comment]
Thanks you @jedi404
I will take a look on your mentioned bugs :)

**Comment by jedi404** [comment]
any further help or testing that you need you can contact me on social
media. my community uses your program very heavily. and we love the work
youve put in. i have an average of roughly 30+ actual video users in a
server of over 100 people. we can test things extensivly if you need the
testers. and we would do this freely just to help make the software we use
run better.  as a further note. the driver support for newer logitech
webcams have been known to have frame rate issues in .8 the issue has gone
away as of .11 but that may be due to windows rather than the plugin. a new
version of windows 10 (the anniversary or right after) changed how webcams
are handled on a driver level. this may be worth looking into.

**Comment by Miscni** [comment]
Nion098 , what GPU do you have?
GPU = Graphiccard
Just want to say, that is you use a Laptop, with AMD/Intel, then please send your Teamspeak Client Log (CTRL + L)
it is the error log, that I was interest in.

**Comment by Nion098** [comment]
Miscni,
 Please tell me what does a graphic card has to do with this,my problem is: it is not showing any errors but when i click right click on any channel, i don't see the ts3video option. And i am using gt 740 and it is working perfect with old teamspeak 3 and teamspeak3 video, and even on my laptop its working perfect with old things.

**Comment by Miscni** [comment]
Actaully your GPU has everything to do with TS3Video, while it is using OpenGL
And also, where is your Clientlog from TS.

**Comment by Nion098** [comment]
You didnt sad, you want the log with the new teamspeak 3 and teamspeak3 video or with what i have not ?

**Comment by Miscni** [comment]
Yes I have asked for the log (CTRL+L)

**Comment by mfreiholz** [comment]
Another idea would be that it is being blocked by some anti-virus software? Hmm

**Comment by jedi404** [comment]
we have this issue alot. with multi screen users. what happens is it opens
on a screen thats not there, a quick fix is to right click the icon on your
task bar and select the option to move the app. that should snap your mouse
to a side of the screen where the program is hiding, and a simple drag will
make it show back up. thats assuming its not frozen

**Comment by Miscni** [comment]
Nion098 came online and got it fixed.
For some reason, the error went away.
Problems was solved for Nion098.

**Comment by mfreiholz** [comment]
@Nion098 your problem sounds more like, that the plugin is not being loaded.
You don't see the "TS3VIDEO" menu in the context menu, right?

**Comment by mfreiholz** [comment]
Btw.. I added a new issue for the  "not visible window problem here: #53

**Comment by Nion098** [comment]
Ok so the problem was fixed, when i update Team-speak 3.1.1 but it wasn't working on 3.0.1 .

**Comment by jedi404** [comment]
i can confirm nion098's client is working again hes one of mine. so far so
good with the new version we will keep you posted if anything else pops up
